setup/base_report_to_printer_mail/odoo/addons/base_report_to_printer_mail

```diff
@@ -0,0 +1 @@
+../../../../base_report_to_printer_mail
```

Comment:
change related to `setup` folder are also normally part of the `[IMP] $module: pre-commit stuff` if you follow the OCA migration guidelines: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0#technical-method-to-migrate-a-module-from-150-to-160-branch